recipe/meta.yaml

```diff
@@ -13,19 +13,24 @@ source:
     - 0001-add-version-var.patch
 
 build:
+  skip: True  # [py==312 or s390x or (py>39 and arm64)]
```

Comment:
You should first update `ephem` to the latest version 4.1.5. It will enable py312 support. I wonder why it wasn't built with the latest py312 build-out automatically.
The selector py>39 and arm64 won't build for py310-py311 and osx-arm64. I think after updating ephem you can rewrite this selector.